metricbeat/docs/modules/http.asciidoc

```diff
@@ -5,8 +5,8 @@ This file is generated! See scripts/docs_collector.py
 [[metricbeat-module-http]]
 == http Module
```

Comment:
Ah, sorry. I just noticed that you changed already to `Http module`. That's perfect.